src/main/resources/plugin.yml

```diff
@@ -34,6 +34,10 @@ commands:
    kaboom:
       description: I wonder...
       permission: extras.kaboom
+   ping:
+      aliases: [delay, ms]
+      description: Displays the delay of your connection to the server in milliseconds
```

Comment:
Is this description too long for the readme?

Reply:
Shortened it to "Gets your ping"